### DISCUSSION TITLE
SDRPlusPlus for Android with RTL-SDR V4

### DISCUSSION THREAD

**Discussion by intuiteON** [AlexandreRouma/SDRPlusPlus#1237] [General]
Hello All and perhaps especially @AlexandreRouma,
I tried SDRPlusPlus on Android with latest nightly build and latest driver (updated for RTL-SDR v4) and it doesn't work, which I guess is somehow normal. Is there any update planned on SDRPlusPlus for it to work with RTL-SDR v4 ? or is there a way to make it work ? or am I missing something ?

**Comment by Tomrm96** [comment]
https://github.com/AlexandreRouma/SDRPlusPlus/issues/1254

I have asked, but no response yet.

**Reply by Tomrm96** [reply]
The reply says no timelines at present as he is busy.

**Reply by intuiteON** [reply]
Hello, it has been fixed ! Thx

**Comment by ebob5030** [comment]
Do you still have this problem? I have been using an RTL-SDR V4 with SDR++ on my Pixel 7 Pro and Android 14 with no issues for months. Do you have a proper OTG cable? Have you tried it with a ¨non-V4¨ RTL-SDR? Or from another vendor?

**Reply by intuiteON** [reply]
Hello, I do not have the problem anymore since last week !

**Reply by intuiteON** [reply]
thx

**Comment by intuiteON** [comment]
Ok now it works !!! I do not know who to thank but thank you.

**Comment by arkangel-dev** [comment]
Yeah its not working for me either. I've also a pixel 7 pro with the V4 variant. Keep getting a flat noise
